sycl/test-e2e/Graph/submission_while_executing.cpp

```diff
@@ -0,0 +1,96 @@
+// REQUIRES: cuda || level_zero, gpu
```

Comment:
cuda support for sycl-graph is not yet available on the xmain branch.

// REQUIRES: cuda || level_zero, gpu 
should be 
// REQUIRES: level_zero, gpu

Reply:
Good catch, thanks. This is fixed.